src/components/layout/StatisticSection.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { AiOutlinePlus } from 'react-icons/ai';
+
+export type ContentType = {
+  title: string;
+  statistic: string;
+};
+
+type StatisticSectionType = {
+  content: ContentType[];
```

Comment:
put this in plural for clearer variable context. e.g. contentItems, statisticItems